recipes/opentelemetry-proto/all/conanfile.py

```diff
@@ -1,43 +1,42 @@
 from conan import ConanFile
-from conan.tools.files import get, copy, save
-from conan.tools.layout import basic_layout
+from conan.tools.files import get, copy
 import os
 
 required_conan_version = ">=1.52.0"
 
+
 class OpenTelemetryProtoConan(ConanFile):
     name = "opentelemetry-proto"
     license = "Apache-2.0"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/open-telemetry/opentelemetry-proto"
     description = "Protobuf definitions for the OpenTelemetry protocol (OTLP)"
-    topics = ("opentelemetry", "telemetry", "otlp")
+    topics = ("opentelemetry", "telemetry", "otlp", "pre-built")
+
+    package_type = "application"
     settings = "os", "arch", "compiler", "build_type"
-    no_copy_source = True
 
     def layout(self):
-        basic_layout(self, src_folder="src")
+        pass
```

Comment:
Why `pass` here? I would expected to have a layout here if it used to work (Or was it broken before?)

Reply:
That was accidental. Will revert.